packages/web/src/scss/components/Card/_CardArtwork.scss

```diff
@@ -0,0 +1,16 @@
+@use '../../tools/dictionaries';
+@use 'theme';
+
+.CardArtwork {
+    grid-area: media;
```

Comment:
as this is shared with the actual Media subcomponent, it would be better to use something more generic? grid-area: illustration? or something

Reply:
I'm unable to come up with anything good. Maybe you will? Illustration doesn't work I think, it's weird to place e.g. a video into the "illustration" region.